test/CollapseSpec.js

```diff
@@ -174,7 +172,7 @@ describe('<Collapse>', () => {
     });
 
     it('Defaults to height', () => {
-      assert.equal(wrapper.instance().collapse.getDimension(), 'height');
+      assert.equal(collapseHelpers.getDimension(), 'height');
```

Comment:
**suggestion:** instead of doing this (which doesn't do anything to check that the logic in the component itself is correct), assert on the rendered classes from the object, i.e. that `.width` is not present: https://github.com/moh12594/react-bootstrap/blob/2329c05768f09104c3c6453b875abb61902c9db8/src/Collapse.js#L236